Cargo.toml

```diff
@@ -179,3 +179,6 @@ debug-assertions = false
 #
 # [patch."https://github.com/servo/<repository>"]
 # <crate> = { path = "/path/to/local/checkout" }
+
+[patch."https://github.com/servo/mozjs"]
+mozjs = { git = "https://github.com/sagudev/mozjs", branch = "no-errorstackclone-patch" }
```

Comment:
We need to land https://github.com/servo/mozjs/pull/473 in order to remove this.